api/routes/user.js

```diff
@@ -1,9 +1,11 @@
-import Express from 'express'
-import { validateToken } from '../middleware/auth'
-import { getUserProfile } from '../controller'
+import Express from "express";
+import { validateToken } from "../middleware/auth";
+import { getUserProfile, ContactAdmin } from "../controller";
 
-const UserRouter = new Express.Router()
+const UserRouter = new Express.Router();
 
-UserRouter.get('/', validateToken, getUserProfile)
+UserRouter.get("/", validateToken, getUserProfile);
 
-export default UserRouter
+UserRouter.post("/contact", ContactAdmin);
```

Comment:
This is not user specific api.

Create a new router `NotificationRouter` and new controller, new post api `/email` and controller method `sendEmail`

Right now we only support sending plain text message later on we can send sms, email with attachment, and different template for different emails.